### PR TITLE
[11.0][IMP]meeting count calculation performance

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Improved performance almost 4x faster.

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
@aheficent I have introduced a few changes. Still, I am not sure if this will introduce a performance boost. We'll need to evaluate.

**Comment by AaronHForgeFlow** [comment]
@jbeficent I noticed of a very better performance when accessing contacts in a very big database. About 4x faster.

Edit: Hold on, I have not updated your changes.

**Comment by AaronHForgeFlow** [comment]
Confirmed the performance improvement for this single computation method.

**Review by JordiBForgeFlow — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

File: crm_meeting_commercial_partner/models/res_partner.py [hunk 1]
```diff
@@ -8,21 +8,22 @@
...
-    def _compute_meeting_count(self):
-        super()._compute_meeting_count()
+    def _compute_meeting_count_commercial_partner(self):
+        event_model = self.env['calendar.event']
         for partner in self:
-            partners = partner + partner.child_ids
-            meetings = self.env['calendar.event'].search(
-                [('partner_ids', 'in', partners.ids)])
-            partner.meeting_count = len(meetings)
+            partner.meeting_count = event_model.search_count(
```
Comment (pedrobaeza) [inline crm_meeting_commercial_partner/models/res_partner.py]: Use `read_group` instead for total performance boost. @mreficent has done it before in for example: https://github.com/OCA/contract/commit/402370f1cbe5cde6b484200f87596e9d803a36af#diff-3dc0d52ea9449a93459ef5b7fc36a060R22

Comment (AaronHForgeFlow) [reply crm_meeting_commercial_partner/models/res_partner.py]: Thanks @pedrobaeza ! It is much faster now! :)

**Pull request merged**